src/workspace/view/newworkspacemodel.cpp

```diff
@@ -106,6 +106,11 @@ void NewWorkspaceModel::validateWorkspaceName()
                          .arg(m_workspaceName);
         return;
     }
+
+    if (m_workspaceName.size() > 24) {
```

Comment:
As @Tantacrul said https://github.com/musescore/MuseScore/issues/11362#issuecomment-1108761914
> I suggest a cap of 24 characters. A user should not be able to enter more than that in the text field.

We should just ignore character input without showing an error message.